### ISSUE TITLE
GT 650M not being recognised as a CUDA GPU

### ISSUE THREAD

**Issue by mattwez378** [kozyilmaz/nheqminer-macos#11]
Hi, I have followed your guide and installed nheqminer on my mid 2012 macbook pro, I think it has installed successfully as far as I can tell, it can run on the CPU fine, but when I run the command for the GPU it says no CUDA compatible GPU found.

	==================== www.nicehash.com ====================
		Equihash CPU&GPU Miner for NiceHash v0.5c
	Thanks to Zcash developers for providing base of the code.
	    Special thanks to tromp, xenoncat and djeZo for providing 
	      optimized CPU and CUDA equihash solvers.
	==================== www.nicehash.com ====================

Setting log level to 2
[22:18:47][0x00007fffb9fc83c0] Using SSE2: YES
[22:18:47][0x00007fffb9fc83c0] Using AVX: YES
[22:18:47][0x00007fffb9fc83c0] Using AVX2: NO
[22:18:48][0x00007fffb9fc83c0] CUDA error 'no CUDA-capable device is detected' in func 'getinfo' line 45

I have bootcamped my macbook and the windows side of things recognises the GPU as a CUDA one but doesnt mine anything with it.......

Any help with this would be greatly appreciated :)

**Comment by kozyilmaz** [comment]
@mattwez378 Can you post the output of this command:

```
$ ./nheqminer-old -ci
```

I am gonna quote my previous statement about graphic cards with compute capability < 5.0 here

“Unfortunately currently best nheqminer GPU solvers supports Compute Capability > 5.0 (you may check the list here https://developer.nvidia.com/cuda-gpus) which means GeForce 9xx series minimum.

I added older Tromp GPU solver builds (nheqminer-old), although they work on older GPU's (and sometimes don't) but the hashrate is not really good.

I will suggest giving an OpenCL miner a try (https://github.com/justvanbloom/silentarmy), @denics recommended this in one of the closed issues.”

Hope that helps.

**Comment by mattwez378** [comment]
Hi kozilmaz, 

I have found that on the windows side the driver just needed updating through nvidia and not device manager.

I have now managed to get the 650M working on OSX too I found that I had not restarted the system since installing CUDA (schoolboy error).

I am currently only getting 3Sols/s on the GPU so it is pretty poor, like you have said, when mining on the CPU I average around 17 which is still pretty poor.

Thankyou for the silentarmy link I will try that again as I couldn't get it to work the first time I tried but now I have a little bit more knowledge of what is required it may work.....
